pmda/rmsf.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
```

Comment:
remove this file from the commit (you can `git revert` the commit that added it)